.env.docker

```diff
@@ -6,4 +6,5 @@ POSTGRESQL_PASSWORD=postgres
 REDIS_URL=redis://redis:6379/1
 
 STATEMENT_TIMEOUT=300000
+SECRET_KEY_BASE=fY+kmWiziK5yJf3hopz9Eha/FSCRYpzMkivn0fsVksbJVVDgTmw01zGUiAdeMus05yg1ULcqjpZ9nGvA6r5BeQ==
```

Comment:
Same as above. Moreover, looking at `docker-entrypoint.sh`, this value will never be replaced?